docs/customizing.htm

```diff
@@ -5149,6 +5149,1577 @@ <h2 id="Changing_the_unit_system">Changing the unit system</h2>
         or they can be rebuilt with the tool:</p>
     <pre class="tty cmd">wee_database weewx.conf --backfill-daily</pre>
 
+    <h1 id="import">Importing observation data</h1>
+
+    <p>Some <span class="code">weewx</span> users will have historical data from another source (eg other weather station software or a manually compiled file) that they wish to import into the <span class="code">weewx</span> database. Such data can, depending upon the source, be imported automatically using the <span class="code">wee_import</span> utility. This section details the use of the <span class="code">wee_import</span> utility.</p>
```

Comment:
The word 'that' in this context is acting as a restrictive pronoun (Says Struck & White, "The lawnmower *that* is in the shed."). You want "which"